lib/Ebook.php

```diff
@@ -572,8 +572,8 @@ protected function GetIndexableText(): string{
 				}
 			}
 
-			$this->_IndexableText .= ' ' . $this->Description;
-			$this->_IndexableText .= ' ' . $this->LongDescription;
+			// Remove diacritics and non-alphanumeric characters
+			$this->_IndexableText = trim(preg_replace('|[^a-zA-Z0-9 ]|ius', ' ', Formatter::RemoveDiacritics($this->_IndexableText)));
```

Comment:
This is not correct, because we want to be able to search for characters including diacritics. For example a search for `capek` should return a listing for `Karel Čapek`. 

You can remove diacritics but keep the actual letters using `Formatter::RemoveDiacritics()`.

Reply:
Sorry, if I'm being dense, but I believe this is working as intended. The code above calls `Formatter::RemoveDiacritics()` before `preg_replace()`. I copied the logic from the existing APCu search:

https://github.com/standardebooks/web/blob/2d220ca008f7f79e8a08947e5068c946f8e96153/lib/Ebook.php#L563

The IndexableText for both of Karel Čapek's books includes:

`karel capek`

A search for either "Čapek" or "capek" works in both the APCu and DB implementations because the query is also converted to "capek".